src/components/views/settings/tabs/user/AppearanceUserSettingsTab.tsx

```diff
@@ -213,15 +214,25 @@ export default class AppearanceUserSettingsTab extends React.Component<IProps, I
         this.setState({customThemeUrl: e.target.value});
     };
 
-    private onLayoutChange = (e: React.ChangeEvent<HTMLInputElement>): void => {
+    /*private onLayoutChange = (e: React.ChangeEvent<HTMLInputElement>): void => {
```

Comment:
Needs to be removed I guess?

Reply:
I need this later. Of course it needs to be changed a little bit but maybe leave it for reference until then?

Reply:
In my view, it's a bad practice to commit commented out chunks like this. We should instead leverage our tools to help us. For this PR, let's remove the unused things. We can easily recover and modify any bits that are needed later via Git / GitHub.

Reply:
Good point, I'll do that